DQMOffline/Lumi/plugins/ZCounting.h

```diff
@@ -113,7 +141,24 @@ class ZCounting: public DQMEDAnalyzer{
   const double MUON_MASS  = 0.105658369;
   const double MUON_BOUND = 0.9;
 
-  // Histograms
+
+  const Double_t ELECTRON_MASS  = 0.000511;
```

Comment:
please use native types (double - or maybe better float?)

Reply:
Float is fine. Changed.